ENG-2465 - Add New Login API responses and fields (#3603)
FusionAuth/fusionauth-site: astro/src/content/docs/apis/_login-id-type-field.mdx

import APIField from 'src/components/api/APIField.astro';
import InlineField from 'src/components/InlineField.astro';
import Aside from 'src/components/Aside.astro';

<APIField name={props.plural ? 'loginIdTypes' : 'loginIdType'} type="String" defaults="email, username if email not found" optional since="1.99.9">

>   Type{props.plural ? 's' : ''} that FusionAuth will attempt to resolve a user identity with, using <InlineField>user.identities\[x].type</InlineField> and <InlineField>loginId</InlineField>. Can be one{props.plural ? ' or more' : ''} of the following:

Author: I still think it could be confusing how we talk about identityType vs. loginIdType. I decided to equate them in the login API request.

>   Type{props.plural ? 's' : ''} that FusionAuth will attempt to resolve a user identity with, using <InlineField>user.identities\[x].type</InlineField> and <InlineField>loginId</InlineField>. Can be one{props.plural ? ' or more' : ''} of the following:

Reviewer: ```suggestion
The identity type{props.plural ? 's' : ''} that FusionAuth will compare the <InlineField>loginId</InlineField> to. Can be one{props.plural ? ' or more' : ''} of the following:
```
